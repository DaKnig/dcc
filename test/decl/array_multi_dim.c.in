int [const 1] [restrict const 2] foo[static 3];
