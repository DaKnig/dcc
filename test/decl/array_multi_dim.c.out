tree representation of array_multi_dim.c.in:
foo is a 
			static array of size 
|   |   |   |   -> (int)3
of:
					const restrict array of size 
|   |   |   -> (int)2
of:
			const array of size 
|   |   -> (int)1
of:
	i16 ;
